vevestaX/vevesta.py

```diff
@@ -201,91 +202,66 @@ def dump(self, techniqueUsed, filename=None, message=None, version=None, showMes
             df_messages.to_excel(writer, sheet_name='messages', index=False)
             pandas.DataFrame(self._data).to_excel(writer,sheet_name='sampledata',index=False)  
 
-        # self._plot(filename)
+        self.__plot(filename)
 
         if showMessage:
-            message = self.getMessage()
+            message = self.__getMessage()
             print(message)
 
-        
-
-
 
-    def _plot(self, filename):
+    def __plot(self, filename):
 
         modelingData = None
 
         if (filename == None):
             return print("Error: Provide the Excel File to plot the models")
 
-        # skipcols = ['experimentID', 'features']
-        # cols = for column in modelingData
-
         if (os.path.isfile(filename)):
             modelingData = pandas.read_excel(filename, sheet_name='modelling', index_col=[])
-
-        modelingData['timestamp in UTC'] = pandas.to_datetime(['timestamp in UTC'],)
-        print(modelingData['timestamp in UTC'].dtypes)
-
         
         # excluding numeric, datetime type
         nonNumericColumns = modelingData.select_dtypes(exclude=['number', 'datetime'])
         modelingData.drop(nonNumericColumns, axis=1, inplace=True)
         modelingData.drop('experimentID', axis=1, inplace=True)
-        print(modelingData)
-        # print(nonNumericColumns)
+        
+        # changing the values in timestamp column to dates (vector slicing)
+        nonNumericColumns['timestamp in UTC'] = nonNumericColumns['timestamp in UTC'].str[:9]
```

Comment:
Dont use str to extract date, use pandas.to_datetime(['timestamp in UTC'] with right timestamp format.